dockerfiles/components/stacks-network.dockerfile

```diff
@@ -0,0 +1,12 @@
+# build stage
+FROM arm64v8/rust:1.67 as builder
+
+COPY . ./
+
+RUN cargo build --release --manifest-path ./components/stacks-network/Cargo.toml
+
+# prod stage
+FROM gcr.io/distroless/cc
+COPY --from=builder target/release/stacks-network /
+
+ENTRYPOINT ["./stacks-network"]
```

Comment:
I'm not familiar with distroless, is this a better approach than what we have in the other dockerfiles?

Reply:
No idea! I've changed to be the same as what we use for other dockerfiles